src/components/BarChart/BarChart.jsx

```diff
@@ -29,7 +31,7 @@ const {
 } = React.PropTypes;
 
 /**
- * {"categories": ["visualizations", "charts"]}
+ * {"categories": ["visualizations", "charts"], "madeFrom": ["ContextMenu", "ToolTip"]}
```

Comment:
I added `ToolTip` here since `Bars` is using it under the hood but it's currently private.